scrape.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import pandas as pd
+from requests_html import HTMLSession
+import re
+
+
+# Events class
+class Events:
```

Comment:
It is better to have a class called "Event"  (singular) then something like an array or a list for "Events" (plural)

Reply:
❤️